IPython/core/completer.py

```diff
@@ -76,6 +76,13 @@
 from IPython.utils.py3compat import builtin_mod, string_types, PY3
 from traitlets import CBool, Enum
 
+try:
+    import jedi
+    import jedi.api.helpers
+    import jedi.parser.user_context
+except ImportError:
+    pass
```

Comment:
From in person discussion: handle the case where Jedi is not importable, by skipping the jedi codepath.